client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/configure-email-notification/email-address-editor.tsx

```diff
@@ -116,16 +135,16 @@ export default function EmailAddressEditor( {
 			return setValidationError( { email: translate( 'Please enter a valid email address.' ) } );
 		}
 		if ( showCodeVerification ) {
-			// TODO: verify email address with code
+			handleVerifyEmail();
 		} else if ( verifiedContacts.emails.includes( emailItem.email ) ) {
-			handleSetEmailItems();
+			handleAddVerifiedEmail();
 		} else {
-			setShowCodeVerification( true );
-			// TODO: implement sending verification code
+			handleSendVerificationCode();
 		}
```

Comment:
[100% non-blocking, stylistic] _This comment uses a mindset borrowed from the Golang community._

To minimize nesting and make the "happiest" code path the one that's least nested, this conditional could be re-written as follows; that is, if y'all like it:

```ts
		if ( showCodeVerification ) {
			handleVerifyEmail();
			return;
		}

		if ( ! verifiedContacts.emails.includes( emailItem.email ) ) {
			handleSendVerificationCode();
			return;
		}

		handleAddVerifiedEmail();
```

Reply:
It is a very good suggestion. Thanks, I have made the changes - https://github.com/Automattic/wp-calypso/pull/77211/commits/d69e610732abd99945aa4e513de12e643d68a341